daemon/module/docker/volumedriver/voldriver.go

```diff
@@ -212,12 +213,14 @@ func (mod *Module) buildMux() *http.ServeMux {
 		var pr pluginRequest
 		if err := json.NewDecoder(r.Body).Decode(&pr); err != nil {
 			http.Error(w, fmt.Sprintf("{\"Error\":\"%s\"}", err.Error()), 500)
+			log.Error(errors.WithError("/VolumeDriver.Create: error decoding json", err))
```

Comment:
Hi @clintonskitson,

These should be `log.WithField("error", err).Error("/VolumeDriver.Create: error decoding json")`. I won't repeat this for the entire file, but it's in numerous places.